### ISSUE TITLE
Encoding problem with `prtvt` variables

### ISSUE THREAD

**Issue by djhurio** [ropensci/essurvey#11]
I have an encoding problem with several `prtvt` variables. For example:

- `prtvtcfi` from the round 7;
- `prtvtbis` from the round 8.

See the examples.

Here the problem is with party name `The Swedish People´s Party (SPP)`.
```
> str(dat$prtvtcfi)
Class 'labelled'  atomic [1:75022] NA NA NA NA NA NA NA NA NA NA ...
  ..- attr(*, "label")= chr "Party voted for in last national election, Finland"
  ..- attr(*, "format.stata")= chr "%10.0g"
  ..- attr(*, "labels")= Named num [1:22] 1 2 3 4 5 6 7 8 9 10 ...
  .. ..- attr(*, "names")=Error in strtrim(encodeString(object, quote = "\"", na.encode = FALSE),  : 
  invalid multibyte string at '<b4>s Par<74>y (SPP)"'

```  

Here the problem is with party names from Iceland.
```
> str(dat$prtvtbis)
Class 'labelled'  atomic [1:75022] NA NA NA NA NA NA NA NA NA NA ...
  ..- attr(*, "label")= chr "Party voted for in last national election, Iceland"
  ..- attr(*, "format.stata")= chr "%12.0g"
  ..- attr(*, "labels")= Named num [1:19] 1 2 3 4 5 6 7 8 9 10 ...
  .. ..- attr(*, "names")=Error in strtrim(encodeString(object, quote = "\"", na.encode = FALSE),  : 
  invalid multibyte string at '<f0>isflo<6b>kinn"'

```
This is not directly related to your package. I have the same problem if I read ESS data from the STATA data files as well. I do not have this problem if I read ESS data from the SPSS data files.

**Comment by cimentadaj** [comment]
Thanks. I can reproduce both errors. I presume this might be a bug from `haven::read_dta` as `ess::ess_rounds` uses `haven::read_dta` to read the data. When I have some time I'm gonna check whether the error persists with `foreign::read.dta`, if so then I'll create an issue in `haven`.

For the next release I've given the user the option to **download** the data in all three formats (SPSS, Stata or SAS). That is, download the data files into your local computer. For example:

`ess_country("Netherlands", 1:2, your_email, only_download = TRUE, output_dir = ".", format = "spss")`

will download the data to the working directory as a `spss` file. A solution could be that if `only_download = FALSE`, the functions read in the data in the format specified in `format`. For example:

`ess_country("Netherlands", 1:2, your_email, only_download = FALSE, output_dir = NULL, format = "spss")`

would read the data using `haven::read_spss`. What do you think? In principle this should not alter the current behaviour of the functions because the default is `stata`. But it's a workaround that gives more control over these type of errors.

**Comment by djhurio** [comment]
I agree, `only_download = FALSE, output_dir = NULL, format = "spss"` would be a good solution.

**Comment by cimentadaj** [comment]
Great. Will add on the to-do list.

**Comment by cimentadaj** [comment]
Filed an issue with `haven` [here](https://github.com/tidyverse/haven/issues/325)

**Comment by cimentadaj** [comment]
Will be available in next release with in commit e8915bcf2b3e37dbbeff39103b36c7cdcea04526.
